client/src/components/UsersList.tsx

```diff
@@ -98,19 +101,31 @@ export const UsersList: FC<IUserListProps> = ({ usersList }) => {
                   <TableCell>
                     <Grid container columnSpacing={1} justifyContent="center">
                       <Grid item>
-                        <IconButton>
-                          <ArticleIcon />
-                        </IconButton>
+                        <Tooltip title="Show user posts">
+                          <IconButton
+                            onClick={() =>
+                              navigate(`/admin/${user.id}/posts`, {
```

Comment:
It would be more readable if it was in a separated function

Reply:
Replaced